scripts/linters/css_linter.py

```diff
@@ -147,8 +147,8 @@ def get_linters(config_path, files_to_lint, verbose_mode_enabled=False):
         verbose_mode_enabled: bool. True if verbose mode is enabled.
 
     Returns:
-        linter: tuple(ThirdPartyCSSLintChecksManager). Returns custom and
-            third_party linter objects.
+        linter: tuple(None, ThirdPartyCSSLintChecksManager). Returns a 2-tuple
```

Comment:
Sorry -- here and below, there is one other thing. You don't need to write "linter: " at the beginning. Please fix the other four cases too and make sure your docstrings are in alignment with other docstrings in the codebase.

Reply:
Done.